.travis.yml

```diff
@@ -45,6 +46,7 @@ matrix:
     env: USE_CMAKE=YES
 
 script:
+  - if [ ${TRAVIS_OS_NAME} = 'linux' ]; then KWStyle -R -v -gcc -d src; fi
```

Comment:
I already configured the KWStyle.xml in a lax way (for me). The tests folder would need a laxer configuration to pass currently. Do you prefer a laxer configuration, see that later, or fix the "problems" reported for tests with the current configuration ? I am in favor of seeing that later to have minimal merges (instead of a big one), but you are the boss.

```
[ -gcc ]
   = Generate errors as GCC format
```